weld.fs.js

```diff
@@ -329,7 +329,7 @@ export const weld = defineFeature(function(context is Context, id is Id, definit
 export function CodeELWeld(context is Context, id is Id, oldDefinition is map, definition is map,
     specifiedParameters is map, hiddenBodies is Query) returns map
 {
-    if (definition.weldType != WeldType.FILLET_WELD && !definition.buttOtherSide)
+    if (definition.weldType != WeldType.FILLET_WELD && definition.weldType2 == oldDefinition.weldType2)
```

Comment:
```suggestion
    if (definition.weldType != WeldType.FILLET_WELD && !specifiedParameters.weldType2)
```
I would suggest this, otherwise `weldType2` will change every time the user changes a parameter without changing `weldType2`